portfolio/src/main/webapp/gallery.html

```diff
@@ -8,117 +8,75 @@
     <script src="script.js"></script>
   </head>
 
-<style>
-div.gallery {
-  margin: 5px;
-  border: 1px solid #ccc;
-  float: left;
-  width: 180px;
-}
-
-div.gallery:hover {
-  border: 1px solid #777;
-}
-
-div.gallery img {
-  width: 100%;
-  height: auto;
-}
-
-div.desc {
-  padding: 15px;
-  text-align: center;
-}
-div.gallery1 {
-  margin: 5px;
-  border: 1px solid #ccc;
-
-  width: 180px;
-}
-
-div.gallery1:hover {
-  border: 1px solid #777;
-}
-
-div.gallery1 img {
-  width: 100%;
-  height: auto;
-}
-
-div.desc {
-  padding: 15px;
-  text-align: center;
-}
-</style>
 </head>
-<body>
-    <div class = "header">
+<body id = "body">
+    <div id = "header">
         <a href="index.html"><i class="material-icons">arrow_back</i></a>     
     </div>
 
-<div class="gallery1">
+<div id ="gallery1">
   <a target="_blank" href="images/avatar.png">
     <img src="images/avatar.png" width="600" height="400">
   </a>
-  <div class="desc">This is me</div>
+  <div id ="desc">This is me</div>
 </div>
 
 <h1>Pets</h1>
-<div class="gallery">
+<div id ="gallery">
   <a target="_blank" href="images/Maya.jpg">
     <img src="images/Maya.jpg" width="600" height="400">
   </a>
-  <div class="desc">Here is my dog Maya.</div>
+  <div id ="desc">Here is my dog Maya.</div>
 </div>
 
-<div class="gallery">
+<div id ="gallery">
```

Comment:
IDs should be unique to a page. So you shouldn't have 2 id's both with "gallery". If you want to apply CSS to two different elements, use 'class' instead.